spec/features/user_can_see_book_show_page_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+describe "When a visitor visits a Book show page" do
+  before(:each) do
+    @user1 = User.create(name: "Lee")
+    @book = Book.create(title: "Eaarth")
+    @review1 = Review.create(body: "Good book", rating: 5, user: @user1)
+    @review2 = Review.create(body: "Ok book", rating: 3, user: @user1)
```

Comment:
You can also create records like this `user: @user` rather than `user_id: @user.id`.  Both work but the second is a little less to type and a little easier to read to.